cub/cub/agent/agent_merge_sort.cuh

```diff
@@ -67,6 +67,7 @@ struct AgentMergeSortPolicy
 
 /// \brief This agent is responsible for the initial in-tile sorting.
 template <typename Policy,
+          bool IS_STABLE,
```

Comment:
When adding API extensions I would strongly prefer if we can get away from raw booleans. 

It is much harder to discern what `true` means in some API call deep in the code, as opposed to `Stability::Stable` or `Stability::Unstable`

That requires us to put a bit more work into the implementation but makes is much easier to work with the API

We have some examples for that here `cub\cub\device\dispatch\tuning\tuning_reduce_by_key.cuh`